src/pages/Home.js

```diff
@@ -0,0 +1,11 @@
+import React, { Component } from 'react'
+
+export class Home extends Component {
```

Comment:
Since you are exporting at the bottom of each component, you will not want to export at the top as this may throw errors.
```suggestion
class Home extends Component {
```